graphdatascience/session/gds_sessions.py

```diff
@@ -106,13 +105,13 @@ def get_or_create(
         if existing_session:
             session_id = existing_session.id
             existing_size = SessionInfo.from_specific_instance_details(existing_session).size
```

Comment:
no need to use the SessionInfo indirection now.
Could directly use `existing_session.memory`

Reply:
Yes, you're right!